intermediate_notebooks/E2E/taxi/NYCTaxi-E2E-GPU-Dask.ipynb

```diff
@@ -0,0 +1,1643 @@
+       "0     8.727488\n",
+       "1    36.659031\n",
+       "2     8.489439\n",
+       "3     4.085510\n",
+       "4     8.614785\n",
+       "Name: prediction, dtype: float32"
+      ]
+     },
+     "execution_count": 30,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# generate predictions on the test set\n",
+    "'''feed X_test as a dask.dataframe'''\n",
+    "prediction = xgb.dask.predict(client, trained_model['booster'], X_test).persist()\n",
+    "\n",
+    "wait(prediction)\n",
+    "\n",
+    "prediction.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(dask.dataframe.core.Series,\n",
+       " dask_cudf.core.Series,\n",
+       " dask_cudf.core.DataFrame,\n",
+       " dask_cudf.core.DataFrame)"
+      ]
+     },
+     "execution_count": 31,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "type(prediction), type(Y_test['fare_amount']), type(X_test), type(Y_test)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pred = dask_cudf.from_dask_dataframe(prediction).reset_index(drop=True)\n",
+    "true = Y_test['fare_amount'].reset_index(drop=True)\n",
+    "\n",
+    "type(true), type(pred)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 39,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0     8.727488\n",
+       "1    36.659031\n",
+       "2     8.489439\n",
+       "3     4.085510\n",
+       "4     8.614785\n",
+       "Name: prediction, dtype: float32"
+      ]
+     },
+     "execution_count": 39,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pred.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 40,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0     9.0\n",
+       "1    41.0\n",
+       "2     9.0\n",
+       "3     4.5\n",
+       "4     9.0\n",
+       "Name: fare_amount, dtype: float32"
+      ]
+     },
+     "execution_count": 40,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "true.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 41,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1.8316325556822064"
+      ]
+     },
+     "execution_count": 41,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Calculate RMSE\n",
+    "squared_error = ((pred-true)**2)\n",
+    "\n",
+    "# compute the actual RMSE over the full test set\n",
+    "math.sqrt(squared_error.mean().compute())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## SAVE THE TRAINED MODEL"
```

Comment:
We used to have more explanatory text here. Is there a reason to remove it?

Reply:
Will add the original code and text.